pages/glossary/large-scale-text.md

```diff
@@ -0,0 +1,9 @@
+---
+title: Large Scale Text
+key: large-scale-text
+---
+
+Text nodes with a [advance height](https://www.w3.org/TR/css-values/#length-advance-measure) of 18 [points](https://www.w3.org/TR/css-values/#pt), or 14 [points](https://www.w3.org/TR/css-values/#pt) and a [font weight](https://www.w3.org/TR/css-fonts-3/#font-weight-prop) is `bold`, `bolder` or a number of 700 or higher.
```

Comment:
"a font weight is" => "**the** font height is"

Also, multiple and/or in the same sentence make it hard to parse. Maybe make a bullet-point list, or use semi-colon to hierarchise vs comma ("18 points; or 14 points and weight either bold, bolder, or 700+").